tests/unit/ml_handlers/test_autogluon.py

```diff
@@ -0,0 +1,138 @@
+import time
+from unittest.mock import patch
+import pandas as pd
+from mindsdb_sql import parse_sql
+
+# PATH ISSUES AGAIN :(
```

Comment:
Could you open a separate issue describe this problem in a bit more detail, then linking its number back in this comment, please?